app/src/pages/inside/launchesPage/launchFiltersToolbar/allLatestDropdown/allLatestDropdown.scss

```diff
@@ -0,0 +1,118 @@
+.all-latest-dropdown {
+  position: relative;
+  height: 30px;
+  background-color: $COLOR--gray-91;
+  cursor: pointer;
+
+  @media (max-width: $SCREEN_SM_MAX) {
+    background-color: $COLOR--gray-80;
+  }
+}
+
+.selected-value {
+  display: flex;
+  align-items: stretch;
+  justify-content: space-between;
+  height: 30px;
+}
+
+.value {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  min-width: 150px;
+  color: $COLOR--charcoal-grey;
+  font-family: $FONT-SEMIBOLD;
+  font-size: 13px;
+  text-align: center;
+  text-transform: uppercase;
+
+  @media (max-width: $SCREEN_SM_MAX) {
+    display: none;
+  }
+}
+
+.value-short {
```

Comment:
![image](https://user-images.githubusercontent.com/31861805/49222292-88567900-f3ec-11e8-9301-9896a443516f.png)
May be we need to add some `padding` here